### ISSUE TITLE
Crashing when opening an old criteria quest?

### ISSUE THREAD

**Issue by BLSK** [joshiejack/Progression#17]
Not sure..was opening it in 3.8 but I was doing a run through of how the tabs were working and then crashing on a quest I made in 3.7.

http://pastebin.com/5WVUEhFL

Just going to delete it and remake it but not sure why it suddenly broke.  I didn't change anything in it since the last time it broke XD  it was the Rare Sapling one I changed to a non-space... was just disabling natura till the player gains a jungle sapling(which tend to be semi rare in biomesOplenty

I guess ill see if other criterias are broke too... hope to understand the issue.

**Comment by BLSK** [comment]
Ok nvm...think I see the issue...dang spaces in another mod....

**Comment by joshiejack** [comment]
If you wouldn't mind, could you test things with 0.4.0
I might just 'cancel' a couple features and release it. In theory it should have fixed some issues, but because of the refactor I'm not sure how stable it is yet.

http://joshiejack.uk:8080/job/Progression/

**Comment by BLSK** [comment]
I just wanted to just state I notce you wrote in this:
I didn't see it at the time after all XD  I am not terribly use to Github's notification system but almost there XD.

Um... I didn't try the space thing yet if that was one of the tests you were getting at?  I did add your Jenkins to my favorites so I could download the builds in the future though.  I have/will still tell ya if something happens...just this one was premature as I saw the space thing after posting. I'll try and break my pack with the 4.0.28 in a hot second(at least things I know)
Thaumic Bench:
Not allowing me to craft things if not unlocked: true
Spacey Objects:
Allow Spacey Names: looks like true XD
Popup:
Where is it: Right spot so true
Buttons?:
Not Crashing maybe with buttons:  Seems true...can't reproduce now...

Don't recall any others but looks like things are working well.
